mne_bids/mne_bids.py

```diff
@@ -641,6 +641,18 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     kind = _handle_kind(raw)
 
     bids_fname = bids_basename + '_%s%s' % (kind, ext)
+
+    # check whether the info provided indicates that the data is emptyroom
+    # data
+    emptyroom = False
+    if subject_id == 'emptyroom' and task == 'noise':
+        emptyroom = True
+        er_date = datetime.fromtimestamp(
+            raw.info['meas_date'][0]).strftime('%Y%m%d')
```

Comment:
wait this is dangerous ... what if `raw.info['meas_date']` is None?

Reply:
yeah, I just realised that as well after pushing that last commit. Will fix.
If it is `None` I'll just assume that the provided value is correct